.gitignore

```diff
@@ -35,3 +35,5 @@ web-ext/
 
 yarn.lock
 .DS_Store
+tmp/
+docs/html/
```

Comment:
I removed this entry in a previous patch: https://github.com/mozilla/addons-linter/commit/2d1c81e8724bce241e234e40d8567a903715b5e8#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L36, why do we need it again here?

Reply:
eh, `docs/html` actually just slipped into this PR by mistake, it wasn't meant to be there, only `tmp/` should have been added to .gitignore.